__tests__/app.test.js

```diff
@@ -0,0 +1,33 @@
+const request = require('supertest')
+const app = require('../app')
+const db = require('../db/connection.js')
+const seed = require('../db/seeds/seed')
+const testData = require('../db/data/test-data')
+
+beforeEach(() => seed(testData))
+afterAll(() => db.end())
+
+
+describe("app", () => {
+    describe("GET /api/topics", () => {
+        test("should return an array of objects with properties of slug and description", () => {
+            return request(app).get('/api/topics').expect(200).then((res) => {
+                expect(res.body.topics[0]).toMatchObject({
+                    slug: expect.any(String),
+                    description: expect.any(String)
+                })
+                expect(typeof res.body.topics).toBe("object")
+                expect(res.body.topics).toHaveLength(3)
+            })
+
+        });
+        test("should return 404 when route does not exist", () => {
```

Comment:
/api/*invalid* will affect all paths, not just /api/topics, it would be better to have this as its own test (e.g. /api/*).